test/unit/util/get-all-keys.js

```diff
@@ -0,0 +1,40 @@
+import keys from '../../../src/util/get-all-keys';
+import createSymbol from '../../../src/util/create-symbol';
+
+describe('utils/get-all-keys:', () => {
+  it('should return normal keys from an object', () =>
+    expect(keys({ foo: 1, bar: 2 }))
+      .to.have.lengthOf(2)
+      .and.to.deep.equal(['foo', 'bar'])
+  );
+
+  it('should return symbol keys from an object', () => {
+    const secret = createSymbol('secret');
+    expect(keys({ foo: 1, [secret]: 2 }))
+      .to.have.lengthOf(2)
+      .and.to.deep.equal(['foo', secret]);
+  });
+
+  describe('when enumerable option is set', () => {
+    const obj = {};
+    beforeEach(() =>
+      Object.defineProperties(obj, {
+        foo: { enumerable: true, value: 1 },
+        bar: { enumerable: false, value: 2 },
+      })
+    );
+
+    describe('to false', () => {
+      it('should not return enumerable keys', () =>
```

Comment:
It's just a bad name for a flag in this case 👅 
Code using this function now, was using either one of those methods. Once I change the `enumerable` flag will change the test for sure 👍